website/src/css/customTheme.scss

```diff
@@ -630,7 +633,7 @@ div[class^="docSidebarContainer"] {
 
     &:after {
       background-size: 1.66rem 1.66rem;
-      margin-right: -6px;
+      // margin-right: -6px;
```

Comment:
This CSS line was responsible for fixing sidebar arrow in the mobile version.

<img width="475" alt="Screenshot 2021-05-06 111736" src="https://user-images.githubusercontent.com/719641/117273883-d04f1500-ae5c-11eb-86d3-746c39282e2c.png">